### PR TITLE
cyborg buff (service)

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
splits serviceborg's shaker into four separate ones; additional reagents to work with.
adds the cooking multitool to serviceborg.

## Changelog
:cl:
add: splits serviceborg's shaker into four separate ones; additional reagents to work with.
add: adds the cooking multitool to serviceborg.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/borg_buffs/code/robot.dm [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+/obj/item/cooking/cyborg/power/examine()
+	. = ..()
+	. += " It's fitted with a [tool_behaviour == TOOL_KNIFE ? "screw" : "bolt"] head."
+
+/obj/item/cooking/cyborg/power/attack_self(mob/user)
+	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, TRUE)
+	if(tool_behaviour != TOOL_ROLLINGPIN)
+		tool_behaviour = TOOL_ROLLINGPIN
+		to_chat(user, "<span class='notice'>You attach the rolling pin bit to [src].</span>")
```
Comment (tf-4) [inline modular_skyrat/modules/borg_buffs/code/robot.dm]:
```suggestion
		to_chat(user, span_notice("You attach the rolling pin bit to [src].")) 
```

File: modular_skyrat/modules/borg_buffs/code/robot.dm [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		icon_state = "rolling_bolt_cyborg"
+	else
+		tool_behaviour = TOOL_KNIFE
+		to_chat(user, "<span class='notice'>You attach the knife bit to [src].</span>")
```
Comment (tf-4) [inline modular_skyrat/modules/borg_buffs/code/robot.dm]:
```suggestion
		to_chat(user, span_notice("You attach the knife bit to [src].")) 
```

**Comment by SpaceVampire** [comment]
I really think that adding the electric welder as upgrade for all borgs is too much of a major buff.
Isn't that just giving every single cyborg fast self-healing? Why would anything but an Engi-Borg even have it?

**Comment by GoldenAlpharex** [comment]
> I really think that adding the electric welder as upgrade for all borgs is too much of a major buff.
> Isn't that just giving every single cyborg fast self-healing? Why would anything but an Engi-Borg even have it?

At first I was agreeing with that PR, but uh, yeah, I agree with this, that's very likely going to absolutely yeet the cyborg balance out of the window.

**Comment by ZephyrTFA** [comment]
Just gonna step in and say that being able to self repair as a borg is very powerful. Giving all models the ability to self repair like the engineering borg, but also do it twice as fast, seems like an absurd buff

**Comment by SpaceVampire** [comment]
I would maybe just keep the stuff for the serviceborg in this one, and uh.
Move the welder changes into another PR, if you still want to go ahead with it.
Given that people generally seem to be cool with the first part, but not the second part.

File: modular_skyrat/modules/borg_buffs/code/robot.dm [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	desc = "A cyborg fitted module resembling the rolling pins and Knifes"
+	icon = 'modular_skyrat/modules/borg_buffs/icons/items_cyborg.dmi'
+	icon_state = "knife_screw_cyborg"
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	toolspeed = 0.5
+
+/obj/item/cooking/cyborg/power/examine()
+	. = ..()
+	. += " It's fitted with a [tool_behaviour == TOOL_KNIFE ? "screw" : "bolt"] head."
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/borg_buffs/code/robot.dm]: This isn't a powerdrill 😔 
```suggestion
	. += " It's fitted with a [tool_behaviour == TOOL_KNIFE ? "knife" : "rolling pin"] head."
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
So close from being done.

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**